packages/devtools/lib/src/timeline/timeline.css

```diff
@@ -55,12 +55,28 @@
     border-color: var(--selected-border);
 }
 
-.flame-chart {
+.flame-chart-container {
     background-color: var(--shaded-background-darker);
     position: relative;
     overflow: hidden;
 }
 
+.timeline-background {
+    background-color: var(--shaded-background-darker);
+    min-width: 100%;
+    position: -webkit-sticky;
```

Comment:
nit: we don't need to add -webkit-sticky. the browser we support handles this feature without a prefix.